combinetf2/tensorwriter.py

```diff
@@ -96,8 +96,8 @@ def add_pseudodata(self, h, name=None, channel="ch0"):
             )
         self.dict_pseudodata[channel][name] = self.get_flat_values(h)
 
-    def add_process(self, h, name, channel="ch0", signal=False):
-        self._check_hist_and_channel(h, channel)
+    def add_process(self, h, name, channel="ch0", signal=False, masked=False):
```

Comment:
General question:  Why not just require the user to always call add_channel before calling add_process? (in this case the "masked" parameter would not be needed here, where it could be confusing/inconsistent)

Reply:
Yes we could be more restrictive and require the user to always call add_channel first. Probably less error prone.